test/utils/trends/data.test.js

```diff
@@ -49,12 +49,12 @@ describe('[trends] data utils', () => {
         expect(utils.findTimeOfDayBin(binSize, 0)).to.equal(1800000);
       });
 
-      it('should assign a bin of `1800000` to a datum at time 3600000', () => {
-        expect(utils.findTimeOfDayBin(binSize, 3600000)).to.equal(1800000);
+      it('should assign a bin of `1800000` to a datum at time 3599999', () => {
+        expect(utils.findTimeOfDayBin(binSize, 3599999)).to.equal(1800000);
```

Comment:
am I reading this wrong?
`findTimeOfDayBin(...)` 
- the size is one hour based one the test constant `binSize`
- in this test we are giving it  a time of `3599999`  
- based on the name of the function I expect to be returned a time
- but in the test we say the the returned value `1800000` is the bin

I know I am likely misreading it but thought I should say it now